content/source/docs/extend/best-practices/depending-on-providers.html.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: "extend"
-page_title: "Extending Terraform: Depending on Providers"
+page_title: "Building Terraform Providers: Depending on Providers"
```

Comment:
This section seems to be missing the subsection titles 😢 

```suggestion
page_title: "Building Terraform Providers - Depending on Providers"
```

Reply:
Subsection titles?

Reply:
You can ignore my comment there, the inclusion of subsections (e.g. Building Terraform Providers - Best Practices - Depending on Providers) is inconsistently applied, but has nothing to do with these changes.